airflow/api_connexion/openapi/v1.yaml

```diff
@@ -1381,11 +1381,198 @@ paths:
               schema:
                 $ref: '#/components/schemas/VersionInfo'
 
+  /login:
+    post:
+      summary: User login
+      description: |
+        Verify user and return a user object and JWT token as well
+      x-openapi-router-controller: airflow.api_connexion.endpoints.user_endpoint
+      operationId: login
+      tags: [User]
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Login'
```

Comment:
This is only ever used here, do we should just "inline" it, rather than making a schema for it.

_If_ we even have parameters for it, rather than just let the auth backend handle it (i.e. the `Authorization` header)